Lesson3/Program.cs

```diff
@@ -28,17 +28,15 @@ public static List<string> GetSortedData(string[] fileData)
         [STAThread]
         public static void Main(string[] clArguments)
         {
-            if (clArguments.Any(a => String.Compare(a, "-json", true) == 0) || clArguments.Any(a => String.Compare(a, "-xml", true) == 0))
+            if (clArguments.Any(a => String.Compare(a, "-xml", true) == 0))
```

Comment:
`Observation`: This really should change since it should have been merged in from the console branch.